ElectronicObserver/Window/Tools/ExpeditionRecordViewer/ExpeditionRecordViewerViewModel.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using CommunityToolkit.Mvvm.Input;
+using ElectronicObserver.Common;
+using ElectronicObserver.Data;
+using ElectronicObserver.Database;
+using ElectronicObserver.Services;
+using ElectronicObserver.Utility;
+using ElectronicObserver.Window.Tools.FleetImageGenerator;
+using ElectronicObserver.Window.Tools.SortieRecordViewer;
+using ElectronicObserverTypes.Serialization.DeckBuilder;
+using Microsoft.EntityFrameworkCore;
+
+namespace ElectronicObserver.Window.Tools.ExpeditionRecordViewer;
+
+public partial class ExpeditionRecordViewerViewModel : WindowViewModelBase
+{
+	private ToolService ToolService { get; }
+	private DataSerializationService DataSerializationService { get; }
+
+	private ElectronicObserverContext Db { get; } = new();
+
+	public ExpeditionRecordViewerTranslationViewModel ExpeditionRecordViewer { get; }
+
+	private static string AllRecords { get; } = "*";
```

Comment:
Could be a const I guess ?

Reply:
I don't like consts because they're fields, rather than properties.
![image](https://github.com/ElectronicObserverEN/ElectronicObserver/assets/40002167/8b14b2bc-cbe4-4efa-9e03-d710447db790)
![image](https://github.com/ElectronicObserverEN/ElectronicObserver/assets/40002167/7c9c4854-06c0-454c-b2ad-8b8060714ad3)
Fields don't have a quick references access and can't be debugged (setting a breakpoint).